### ISSUE TITLE
Name conflict with module already in PSGallery

### ISSUE THREAD

**Issue by MartinSGill** [Microsoft/Docker-PowerShell#191]
Name conflict with module already in PSGallery, which might lead to some confusion. https://www.powershellgallery.com/packages/Docker

Perhaps changing the name of the module to `Docker.Powershell` just like the repo would be a good idea.

**Comment by johlju** [comment]
I agree, I would like to see this in the PowerShell Gallery, especially now when it can be release with an '-alpha1' tag in the version number.
It would be more useful having this in the PowerShell Gallery than downloading directly from the dev (master) AppVeyor branch nuget store. Even so when there are only one branch. A faulty build could potentially make a pipeline fail if this is used in a pipeline.
